drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -89,12 +99,30 @@ function DragNDropTemplates(url_name) {
             style['outline-color'] = item.color;
         }
         if (item.is_placed) {
-            style.left = item.x_percent + "%";
-            style.top = item.y_percent + "%";
-            if (item.widthPercent) {
+            if (item.zone_align === 'none') {
+                style.left = item.x_percent + "%";
+                style.top = item.y_percent + "%";
+            } else {
+                // Allow for the input + button width for aligned items
+                if (item.input) {
+                    style.marginRight = '190px';
+                }
+            }
+            if (item.zone_align !== 'none') {
```

Comment:
@bradenmacdonald I don't think we need a separate `if` block here. Could you add the statements belonging to its body to the `else` block above?